.github/workflows/pr-closed.yaml

```diff
@@ -15,18 +15,10 @@ jobs:
       AWS_ACCESS_KEY_ID: ${{ secrets.SITE_DEPLOYER_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.SITE_DEPLOYER_ACCESS_KEY_SECRET }}
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      BUCKET: ${{ format('vapor-website-pulls-pr{0}', github.event.number) }}
 
     steps:
-      - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@v4
-        if: ${{ github.event.pull_request.head.repo.full_name == 'vapor/website' && github.actor != 'dependabot[bot]' }}
-        with:
-          aws-access-key-id: ${{ secrets.SITE_DEPLOYER_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.SITE_DEPLOYER_ACCESS_KEY_SECRET }}
-          aws-region: eu-west-2
       - name: Delete Website Bucket
-        uses: brokenhandsio/s3-website-https-pr-action@1.0.0
+        uses: brokenhandsio/s3-website-pr-action@v2
         with:
           bucket-prefix: "vapor-website-pulls"
           bucket-region: "eu-west-2"
```

Comment:
It appears this needs the `env:`s too still?
I took a look at the implementation of the action and can't say I understood much from that js.

Also, this can be trivially done with the AWS CLI, but I'm fine with keeping things under one action.

For presentation purposes:

```bash
curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
unzip awscliv2.zip
./aws/install

aws s3api delete-bucket --bucket "vapor-design-pulls-pr${{ github.event.number }}"
```
I reckon `"vapor-design-pulls-pr${{ github.event.number }}"` might not be accurate. We'll need to know what format that action uses for bucket names.

Reply:
We can delete the bucket easily but it doesn't delete the deployment in GH. The action does both for you

The env vars are passed in at the top right?

Reply:
Do env vars propagate to actions though?

Reply:
It seems like it's working for 121 and the deployment isn't there anymore?
Though i thought env vars don't just propagate to actions without you using `inherit` or passing them.

OR maybe that was for secrets only?

Reply:
Yeah secrets can be requested by actions if you set the inherit flag but if you set an env var then that's propagated to the environment your action runs in so we're all good. 

121s deployment is not there anymore because the PR was closed so it got cleaned up